schemas/parquet_output_operator_condition.py

```diff
@@ -49,6 +49,8 @@
         Required("reason"): bytesTypeValidator,
         Required("raw_message"): bytesTypeValidator,
         Required("archive_path"): pathToCephInBytesValidator,
+        Required("latest_transition_time"): datetime.datetime,
```

Comment:
Is it really required tho? We are handling also the case when we don't receive one https://gitlab.cee.redhat.com/ccx/parquet-factory/-/merge_requests/90/diffs#3af9ea9b6a758ae7909c1ec88b164ed9bb4ef4d2_61_66